src/core/qgsvectorlayerfeaturecounter.h

```diff
@@ -18,6 +18,8 @@
 #include "qgsvectorlayerfeatureiterator.h"
 #include "qgsrenderer.h"
 #include "qgstaskmanager.h"
+#include "qgsfeature.h"
```

Comment:
```suggestion
```

Reply:
I guess we'll debate the implementation of the featureIds in the subsequent PR. :call_me_hand: